cpp_modules/src/sc2_state/Sc2State.h

```diff
@@ -123,20 +130,30 @@ namespace Sc2 {
 
         std::vector<Action> getLegalActions() const;
 
-        int getValue() const { return mineralGainedPerTimestep() + vespeneGainedPerTimestep(); }
-        std::vector<int> &getOccupiedWorkerTimers() { return _occupiedWorkerTimers; };
+        int getValue() const { return mineralGainedPerTimestep() + 1.5 * vespeneGainedPerTimestep(); }
```

Comment:
Why is vespeneGainedPerTimestep multiplied by 1.5 here?

Reply:
To give some incentive to creating vespene geysers, instead of just workers and bases. 
Without it, vespene collectors become very rare.